catalog/redis-bucket/README.md

```diff
@@ -1,4 +1,74 @@
-# Google Cloud Redis and Storage bucket blueprint
+# Redis Bucket blueprint
```

Comment:
Not sure where "Google Cloud Redis" came from. I think the product name is "Memorystore for Redis". "Redis Bucket" isn't really an improvement. 

What should this actually be named? "Memorystore for Redis with Cloud Storage" is kinda long... maybe just "Memorystore for Redis" ?

Reply:
How about `Google Cloud Storage and Memorystore for Redis`? It's wordy but the most descriptive. I think I would be surprised for a GCS bucket to show up in a blueprint just called `Memorystore for Redis`.

Reply:
sold.